### PR TITLE
Quickstart tests

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: translate/cloud-client/quickstart_test.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from google.cloud import translate
+import mock
+import pytest
+
+import quickstart
+
+
+@pytest.fixture
+def mock_client(cloud_config):
+    original_client_ctor = translate.Client
```
Comment (jerjou) [inline translate/cloud-client/quickstart_test.py]: what's a ctor?

File: vision/cloud-client/quickstart.py [hunk 2]
```diff
@@ -26,11 +27,16 @@ def run_quickstart():
...
+    file_name = os.path.join(
+        os.path.dirname(__file__),
+        'resources/wakeupcat.jpg')
 
     # Loads the image into memory
-    with io.open(fileName, 'rb') as image_file:
-        image = vision_client.image(content=image_file.read())
+    with io.open(file_name, 'rb') as image_file:
+        content = image_file.read()
+        import pdb; pdb.set_trace()
```
Comment (jerjou) [inline vision/cloud-client/quickstart.py]: this looks vestigial

File: translate/cloud-client/quickstart_test.py (hunk 1 above)
Comment (jonparrott) [inline translate/cloud-client/quickstart_test.py]: constructor.

File: vision/cloud-client/quickstart.py (hunk 2 above)
Comment (jonparrott) [inline vision/cloud-client/quickstart.py]: Indeedly-doo. Fixed.

**Comment by jonparrott** [comment]
I'm merging this, if anyone has concerns I'll address them in a follow up PR.

**Comment by jerjou** [comment]
LGTM

**Pull request merged**